doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -801,6 +801,9 @@ Modem libraries
     * The ``CONFIG_LTE_NETWORK_USE_FALLBACK`` Kconfig option.
       Use the :kconfig:option:`CONFIG_LTE_NETWORK_MODE_LTE_M_NBIOT` or :kconfig:option:`CONFIG_LTE_NETWORK_MODE_LTE_M_NBIOT_GPS` Kconfig option instead.
       In addition, you can control the priority between LTE-M and NB-IoT using the :kconfig:option:`CONFIG_LTE_MODE_PREFERENCE` Kconfig option.
+    * ``lte_lc`` tests.
```

Comment:
I don't think it's necessary to add these changes in the changelog. They are transparent to the user and they don't influence the API.

Reply:
I haven't added comments on the tests into the changelog in general. Now that we are removing a test set, I wanted to let users know just in case somebody had run them or wonders why they are removed. So from that perspective, they are not fully transparent.
Those users (that may well not exist) can find information that the removal was intentional and the test coverage didn't actually go down.

Reply:
If I'm the only one who thinks changelog entry is not useful, I'll remove it.
@peknis can comment from doc team point of view?

Reply:
Maybe we could, instead of mentioning the removal of lte_lc tests (as it is not in the public API), mention in the Updated part that the lte_lc_api tests have been extended.

Reply:
We've extended `lte_lc_api` tests every now and then (and same for other tests), and I'm not sure if that's worth mentioning.
If removal of test set is not worth mentioning, I'll just remove this bullet.